### DISCUSSION TITLE
Tips for WayBig + IAFD Match when Part # in title?

### DISCUSSION THREAD

**Discussion by jocklover** [CodyBerenson/PGMA-Modernized#140] [Q&A] [answered]
I have been utilizing the ability to name files w/ the Scene Title to get WayBig match and then IAFD match for the all metadata now being scrapped (vs. listing entire WayBig article title).
However, I am running into issues when the Scene Title includes "Part #".
If I include "Part" as is often listed on WayBig, PLEX stacks the titles. If I remove "Part" and just list the #, PGMA does not correctly match the title. Instead PGMA matches to one of the parts of the title, but not necessarily the correct one.

To be clear, I want the Part specific metadata returned, not general Title metadata.
I am listing this in Discussions as I'm not sure its a "bug". @JPH71 if you think this might be resolved w/ the upcoming updates, I await them! Otherwise, I am open to suggestions to try w/ the naming conventions.

Title: (BROMO) - Betrayed 3 (2016).mp4 
Correct WayBig URL: https://www.waybig.com/blog/2016/01/29/bromo-vadim-black-wesley-woods-betrayed/
URL matched in Log: https://www.waybig.com/blog/2016/02/03/vadim-black-roman-todd-john-delta-wesley-woods-and-leon-lewis-bareback-orgy-in-betrayed-at-bromo/ (**This is Betrayed _4_**)

Log file: [WayBig Titles with Parts.txt](https://github.com/CodyBerenson/PGMA-Modernized/files/8236830/WayBig.Titles.with.Parts.txt)

**Comment by jocklover** [comment, marked as answer]
Hmm, seems if I keep each Part in separate subfolders PLEX does not stack them, even if they are named Part #.
I'll mark this as answered but feel free to chime in if you have other suggetions.

**Comment by JPH71** [comment]
I think all stacked files should be in the same folder.
I keep all stacked files and scenes, extras in the same folder...

**Comment by JPH71** [comment]
Another thing - all the parts must have the same extension...
Try using MKVToolNIX Gui to join up your parts into one file
Start with the first part and append files to it
